### PR TITLE
feat: org page

### PR DESCRIPTION
**Pull request by ArshdeepGrover**
## Type
- ( ) Fix
- ( ) Refactor
- ( ) Style
- ( ) Docs
- (X) Feat
- ( ) Hotfix

### CODE REVIEW

File: apps/commudle-admin/src/app/app-shared-components/event-medium-card/event-medium-card.component.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  @Input() width: string;
+  moment = moment;
+
+  faUserGroup = faUserGroup;
+  faMapPin = faMapPin;
+
+  constructor() {}
+
+  ngOnInit(): void {
+    console.log(this.event);
```
Comment (arpansac) [inline apps/commudle-admin/src/app/app-shared-components/event-medium-card/event-medium-card.component.ts:37]: Remove log

File: apps/commudle-admin/src/app/feature-modules/public-community-groups/components/community-group-activity/community-group-activity.component.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  }
+
+  getPreviousEvents() {
+    this.page_info.start_cursor = '';
+    this.getEvents();
+  }
+
+  getNextEvents() {
+    this.page_info.end_cursor = '';
+    this.getEvents();
```
Comment (arpansac) [inline apps/commudle-admin/src/app/feature-modules/public-community-groups/components/community-group-activity/community-group-activity.component.ts:81]: should the start_cursor and end_cursor be number fields?

File: apps/commudle-admin/src/app/feature-modules/public-community-groups/components/community-group-events/community-group-events.component.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      this.communityGroupsService
+        .pEvents(this.community_group_id, this.limit, '', this.upcomingPageInfo?.end_cursor, 'future')
+        .subscribe((data) => {
+          this.UpcomingEvents = this.UpcomingEvents.concat(data.page.reduce((acc, value) => [...acc, value.data], []));
+          this.upcomingPageInfo = data.page_info;
+          this.isLoading = false;
+        }),
+    );
+  }
+}
```
Comment (arpansac) [inline apps/commudle-admin/src/app/feature-modules/public-community-groups/components/community-group-events/community-group-events.component.ts:61]: Let's change from this.subscriptions.push to the other one which you shared @ArshdeepGrover

File: apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.ts [hunk 4]
```diff
@@ -22,6 +28,7 @@ export class MembershipToggleComponent implements OnInit {
   @Input() shapes: NbComponentShape = 'rectangle';
   @Input() status: NbComponentStatus = 'basic';
   @Input() size: NbComponentSize = 'small';
+  @Input() appearance: NbButtonAppearance = 'filled';
```
Comment (arpansac) [inline apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.ts:31]: Let's define a common enum for different sizes somewhere in the models and call it everywhere.

**Review by arpansac — APPROVED** [review]

File: apps/commudle-admin/src/app/feature-modules/public-community-groups/components/community-group-activity/community-group-activity.component.ts (hunk 2 above)
Comment (ArshdeepGrover) [reply apps/commudle-admin/src/app/feature-modules/public-community-groups/components/community-group-activity/community-group-activity.component.ts:81]: nope, it's string
It should be similar to "WyIyMDIzLTA1LTEwVDIxOjUzOjAwLjAwMFoiLDcwXQ=="

**Pull request merged**